app/src/main/java/de/tum/in/tumcampusapp/component/other/settings/SettingsFragment.java

```diff
@@ -101,6 +111,34 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         view.setBackgroundColor(Color.WHITE);
     }
 
+    private void initCafeteriaCardSelections() {
+        CafeteriaLocalRepository repository = new CafeteriaLocalRepository(TcaDb.getInstance(getContext()));
+        List<Cafeteria> cafeterias = repository.getAllCafeteriasNow();
+        Collections.sort(cafeterias, Cafeteria::compareTo);
```

Comment:
This would sort the cafeterias based on the distance to the user. I think it would make more sense to sort them alphabetically, making it easier for the user to find their cafeteria. What do you think?